src/sidebar/components/test/tag-editor-test.js

```diff
@@ -61,6 +61,10 @@ describe('TagEditor', function() {
   function selectOptionViaDelimiter(wrapper) {
     wrapper.find('input').simulate('keydown', { key: ',' });
   }
+  // Simulates typing ","
```

Comment:
Does it? :) Looks like a copy-paste issue; I'd also suggest this comment is unnecessary...